tap/profile.go

```diff
@@ -27,6 +34,17 @@ type Profile struct {
 	SSOOnlyForRegisteredUsers bool              `bson:"SSOOnlyForRegisteredUsers" json:"SSOOnlyForRegisteredUsers"`
 }
 
+func (p Profile) TableName() string {
+	return ProfilesCollectionName
+}
+
+func (p Profile) DBID() apidef.ObjectId {
```

Comment:
Why it always returns empty ID?

Reply:
There is no db id field inside the struct so it returns empty

Reply:
How it gets stored in SQL, what is the table stucture?